core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1055,28 +1055,28 @@ public MetricKey build() {
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_TRY_REMOVE_COUNT =
       new Builder("Worker.BlockRemoverTryRemoveCount")
-          .setDescription("The total number of blocks tried to be removed from this worker "
-              + "by asynchronous block remover.")
+          .setDescription("The total number of blocks this worker attempted to remove "
+              + "with asynchronous block remover.")
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_REMOVED_COUNT =
-      new Builder("Worker.BlockRemoverBlocksToRemovedCount")
-          .setDescription("The total number of blocks removed from this worker "
+      new Builder("Worker.BlockRemoverBlocksRemovedCount")
+          .setDescription("The total number of blocks successfully removed from this worker "
               + "by asynchronous block remover.")
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_TRY_REMOVE_BLOCKS_SIZE =
       new Builder("Worker.BlockRemoverTryRemoveBlocksSize")
-          .setDescription("The size of blocks to be removed from this worker "
+          .setDescription("The number of blocks to be removed from this worker at this moment "
               + "by asynchronous block remover.")
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_REMOVING_BLOCKS_SIZE =
       new Builder("Worker.BlockRemoverRemovingBlocksSize")
-          .setDescription("The size of blocks is removing from this worker "
+          .setDescription("The size of blocks is being removed from this worker at this moment "
```

Comment:
Can you elaborate what is the meaning of `at this moment`?

Reply:
@HelloHorizon This metric is a gauge which means the value is checked only when queried. When you poke the worker metrics and see a value from this metric, that means the value "at this moment".